src/agentlab/agents/step_agent/agent.py

```diff
@@ -0,0 +1,57 @@
+from typing import List
+
+
+class Agent:
```

Comment:
[nit] I noticed `Agent` is overloaded since  there is a class called Agent in `browsergym.experiments.agent.` We could potentially rename this as `BaseAgent` to avoid conflicts.